### ISSUE TITLE
u8 values not displayed properly

### ISSUE THREAD

**Issue by Tyilo** [kaitai-io/kaitai_struct_visualizer#1]
```
$ xxd foo
00000000: ffff ffff ffff ffff                      ........
$ cat foo.ksy
meta:
  id: foo
  application: foo
  endian: le
seq:
  - id: foo
    type: u8
```

When expanding the tree of `ksv foo foo.ksy`:

```
[-] [root]                    00000000: ff ff ff ff ff ff ff ff                         | ........        
  [-] @foo
    [?] &
    [?] *
    [?] +
    [?] -
    [?] /
    [?] ^
    [?] |
    [?] ~
    [?] **
    [?] <<
    [?] >>
    [?] []
    [?] size
    [?] to_f
    [?] bit_length
```

**Comment by GreyCat** [comment]
Thanks! Actually, it was quite elusive big :) Will fix it now.

**Comment by GreyCat** [comment]
Ok, here you go:

```
[-] [root]                             00000000: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff | ................
  [.] @foo = 18446744073709551615      00000010:                                                 |                 
```

The fix is available in github repository - you can install it manually by something like:

```shell
git clone https://github.com/kaitai-io/kaitai_struct_visualizer
cd kaitai_struct_visualizer
gem build *.gemspec
gem install *.gem
```

A bit later probably it will be released as 0.3.1 in public gem repositories.
